src/datachain/data_storage/sqlite.py

```diff
@@ -643,11 +643,14 @@ def get_dataset_sources(
         self, dataset: DatasetRecord, version: int
     ) -> list[StorageURI]:
         dr = self.dataset_rows(dataset, version)
-        query = dr.select(dr.c.source).distinct()
+        query = dr.select(dr.c.file__source).distinct()
```

Comment:
Another quick fix to make legacy code still working, but will need to be refactor completely in near future